forum-qualitative-social-research.csl

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
-  <!-- Polyglot; journal publishes in English, German, and (maybe?) Spanish -->
```

Comment:
@POBrien333, I assume you didn't remove this on purpose?

Reply:
Sorry. Forgot they. Will add back in.
Any idea iwth the questions, I had, @rmzelle ?